src/mozilla_sec_eia/models/sec10k/ex_21/inference.py

```diff
@@ -275,21 +295,31 @@ def extract_filings(
 
         logits = []
         predictions = []
+        # hidden_states = []
         all_output_df = Ex21CompanyOwnership.example(size=0)
         extraction_metadata = Sec10kExtractionMetadata.example(size=0)
-        for logit, pred, output_df in pipe(_get_data(dataset)):
+        for output_dict in pipe(_get_data(dataset)):
             # TODO: logits and predictions are useful for debugging, do something with them?
-            logits.append(logit)
-            predictions.append(pred)
+            logits.append(output_dict["logits"])
+            predictions.append(output_dict["predictions"])
```

Comment:
I'm not actually using `logits` and `predictions` in the model (yet) but it's useful for debugging.